homeassistant/components/deconz.py

```diff
@@ -0,0 +1,134 @@
+"""Platform integrating Deconz support.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/deconz/
+"""
+
+import asyncio
+import logging
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_API_KEY, CONF_HOST, CONF_PASSWORD, CONF_PORT, CONF_USERNAME,
+    EVENT_HOMEASSISTANT_STOP)
+from homeassistant.core import callback, EventOrigin
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers import discovery
+from homeassistant.util import slugify
+from homeassistant.util.json import load_json, save_json
+
+REQUIREMENTS = ['pydeconz==10']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'deconz'
+
+DECONZ_DATA = 'deconz_data'
+CONFIG_FILE = 'deconz.conf'
```

Comment:
I think the path to the config file should be configurable.

Reply:
This is just the name, the read_json, load_json gets the full path from HASS. From my understanding this is standard behaviour.

Reply:
As, I see what that file is for.

Reply:
If it's not meant for users to manually read/write it, prefix the name with a `.`